File: C++/Iniciante/1046.cpp
#include<iostream>
#include<iomanip>
#include<cmath>

using namespace std;

int main (){
    
    int i, f, t;
    cin >> i >> f;
    if(i>f){
          t=24-i+f;
    }else{
          t=f-i;
    }
    cout << "O JOGO DUROU " << t << " HORA(S)" << endl;
    
	system("pause");    
	return 0;    
}

File: Python3/Iniciante/1046.py
# -*- encoding: utf-8 -*-

INPUT = input()

INICIO = int(INPUT.split()[0])
FIM = int(INPUT.split()[1])

if (INICIO >= FIM):
    TEMPO = (FIM + 24 - INICIO)

else:
    TEMPO = FIM - INICIO

print("O JOGO DUROU %d HORA(S)" % TEMPO)
